pkg/autoops/storage/v2/auto_ops_rule.go

```diff
@@ -154,13 +152,22 @@ func (s *autoOpsRuleStorage) GetAutoOpsRule(
 
 func (s *autoOpsRuleStorage) ListAutoOpsRules(
 	ctx context.Context,
-	whereParts []mysql.WherePart,
-	orders []*mysql.Order,
-	limit, offset int,
+	options *mysql.ListOptions,
 ) ([]*proto.AutoOpsRule, int, error) {
+	var whereParts []mysql.WherePart = []mysql.WherePart{}
+	var orderBySQL string = ""
```

Comment:
Zero value of string and int are "" and 0 already, so I think there won't be necessary to declare the value here! But if you want it to be clear for everyone, I think it's ok to keep it, I want to know your opinion 😅